design-book/src/pane-layout.md

```diff
@@ -0,0 +1,21 @@
+# Pane Layout
+
+## Order of Importance
+
+- Generally, the most important fields/or the must used, should be at the top, quickly accessible. Less important widgets should be placed lower or in sub-panels
+- If every setting in a sub-panel is predicated on something being enabled, the toggle should be in the header.
+
+## Enums
+
+- If a enum is more than 3 items, it should be a drop down.
+- If the field's can be represented as icons then this can be bypassed.
+
+## Sub-Panels
+
+- Should be used when multiple options/buttons/fields are all related and a label would help add context to them.
```

Comment:
This section needs rewording for grammar :)

Reply:
i think https://github.com/bevyengine/bevy_editor_prototypes/pull/55/commits/e1a506428f70bdcfa7be13954e32d55d6ee195ad
addresses this? idk wasnt sure best reword for it.